packages/Webkul/DataTransfer/src/Console/JobExecuteCommand.php

```diff
@@ -0,0 +1,303 @@
+use Webkul\DataTransfer\Helpers\Export;
+use Webkul\DataTransfer\Jobs\Export\ExportTrackBatch;
+use Webkul\DataTransfer\Jobs\Import\ImportTrackBatch;
+use Webkul\DataTransfer\Queue\Worker;
+use Webkul\DataTransfer\Repositories\JobInstancesRepository;
+use Webkul\DataTransfer\Repositories\JobTrackRepository;
+use Webkul\User\Repositories\AdminRepository;
+
+use function Termwind\terminal;
+
+class JobExecuteCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'unopim:queue:work 
+                            {jobId} 
+                            {userEmailId}
+                            {connection? : The name of the queue connection to work}
+                            {--queue= : The names of the queue to work}
+                            {--name=single : The name of the worker}
+                            {--delay=0 : The number of seconds to delay failed jobs (Deprecated)}
+                            {--backoff=0 : The number of seconds to wait before retrying a job that encountered an uncaught exception}
+                            {--memory=128 : The memory limit in megabytes}
+                            {--timeout=60 : The number of seconds a child process can run}
+                            {--tries=1 : Number of times to attempt a job before logging it failed}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Start processing jobs on the queue as a daemon by job id';
+
+    /**
+     * Holds the start time of the last processed job, if any.
+     *
+     * @var float|null
+     */
+    protected $latestStartedAt;
+
+    public function __construct(
+        protected Worker $worker,
+        protected Cache $cache,
+        protected JobInstancesRepository $jobInstancesRepository,
+        protected JobTrackRepository $jobTrackRepository,
+        protected AdminRepository $adminRepository,
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $jobId = $this->argument('jobId');
+        $userId = null;
+
+        if ($this->argument('userEmailId')) {
+            $user = $this->adminRepository->findByField('email', $this->argument('userEmailId'))->first();
+            if (! $user) {
+                $this->error('User not found given user email Id.');
+
+                return 1;
+            }
+
+            $userId = $user->id;
+        }
+
+        $jobInstance = $this->jobInstancesRepository->find($jobId);
+        if (! $jobInstance) {
+            $this->error('Job not found given jobId.');
+
+            return 1;
+        }
+
+        try {
+            $queueName = $this->dispatchJob($jobInstance, $userId);
+
+            $this->listenForEvents();
+
+            $connectionName = $this->argument('connection')
```

Comment:
Either add in a single line or index this 
```php
            $connectionName = $this->argument('connection')
                ?: $this->laravel['config']['queue.default'];
```php

or 
```php
            $connectionName = $this->argument('connection') ?: $this->laravel['config']['queue.default'];
```php

Reply:
resolved